lang/src/org/partiql/lang/ast/ast.kt

```diff
@@ -20,6 +20,7 @@ import org.partiql.lang.util.interruptibleMap
 import org.partiql.lang.util.stringValue
 import java.util.Arrays
 
+@Deprecated("Please use PIG-generated AST instead. ")
```

Comment:
Not pretty sure if this line is necessary.

Reply:
For this and the other one in `ast.kt`: just curious, what makes you unsure?

Reply:
Honestly, I'd consider adding this to every type in this file.

Reply:
They should be necessary. We are purposely deprecating `ExprNode` in favor of PIG-generated `PartiqlAst`.

Same applies to L66.

Reply:
Functions too.

Really, anything not `private`.

Reply:
Of course, if you do this you'll have to evaluate all of the new compiler warnings and add an appropriate `@Supppress` attribute to avoid useless noise of compiler warnings.